src/App.jsx

```diff
@@ -1,33 +1,56 @@
+
+import { useState } from 'react';
+
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
-export const App = () => (
-  <div className="page">
-    <div className="page-content">
-      <div className="box">
-        <div className="field">
-          {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
-          <label htmlFor="search-query" className="label">
-            Search movie
-          </label>
-
-          <div className="control">
-            <input
-              type="text"
-              id="search-query"
-              className="input"
-              placeholder="Type search word"
-            />
+function filterMovies(movies, filterField) {
+  const copyMovies = [...movies];
+  const lowerFilter = filterField.toLowerCase().trim();
+
+  if (filterField) {
+    return copyMovies.filter(
+      movie => movie.title.toLowerCase().trim().includes(lowerFilter)
+        || movie.description.toLowerCase().trim().includes(lowerFilter),
+    );
+  }
+
+  return copyMovies;
+}
+
+export const App = () => {
+  const [searchField, setSearchField] = useState('');
+
+  const visibleMovies = filterMovies(moviesFromServer, searchField);
+
+  return (
+    <div className="page">
+      <div className="page-content">
+        <div className="box">
+          <div className="field">
+            {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
+            <label htmlFor="search-query" className="label">
+              Search movie
+            </label>
+
+            <div className="control">
+              <input
+                type="text"
+                id="search-query"
+                className="input"
+                placeholder="Type search word"
+                onChange={e => setSearchField(e.target.value)}
```

Comment:
it is bad practise to make anonymous arrow function in jsx.
make handler and pass reference here
```
const onChange = e => setSearchField(e.target.value);
```
```suggestion
                onChange={onChange}
```